app/features/settings/all-accounts.tsx

```diff
@@ -76,9 +76,19 @@ export default function AllAccounts() {
         {accounts.map((account) => (
           <SettingsNavButton key={account.name} to="/settings/accounts/123">
             <p>{account.name}</p>
-            <p>{formatUnit(account.usdBalance, 'cent')}</p>
+            <p>
+              {new Money({
+                amount: account.usdBalance,
+                currency: 'USD',
+              }).toLocaleString()}
+            </p>
             {account.satBalance && (
-              <p>{formatUnit(account.satBalance, 'sat')}</p>
+              <p>
+                {new Money({
+                  amount: account.satBalance,
```

Comment:
if this is called sat balance, is the number in btc unit or sat unit? if the latter then we also need pass `unit: 'sat'` to the money constructor

Reply:
yea you're right. I could change this, but I think its fines as a placeholder for now. We will need to pass in accounts here and it should be the same data type that is currently defined in account-selector, then that will change more things